src/components/legend/legend.scss

```diff
@@ -0,0 +1,48 @@
+@use "../../codap/components/vars.scss";
+
+.legend-component {
```

Comment:
I think this overlaps with the legend.scss within the codap folder. I should clean this up so only the size legend styles are configured here.